src/modules/assessment/resolvers/assessment.resolver.ts

```diff
@@ -48,12 +49,14 @@ export class AssessmentResolver {
         return this.assessmentService.getQuestionnaireAssessment(assessment.questionnaireAssessmentId);
     }
 
+    @isPublic(true)
```

Comment:
I would recommend instead of addition this @isPublic annotation, to create a new resolver beside the `assessment.resolver.ts` resolver, and call it `assement-public.resolver.ts` where this resolver will simply not apply the `GqlAuthGuard` and `PermissionGuard`.

In my opinion by by-passing auth check in existing guard using isPublic annotation may be misused later on, and is not exactly necessary to implement a public endpoint.